packages/client/src/components/ClientApp.svelte

```diff
@@ -86,7 +89,10 @@
     if (get(builderStore).inBuilder) {
       builderStore.actions.notifyLoaded()
     } else {
-      builderStore.actions.analyticsPing({ source: "app" })
+      builderStore.actions.analyticsPing({
+        source: "app",
+        ...{ embedded: $appStore.embedded || undefined },
```

Comment:
NAB: I don't think the spread here does anything (and we could just do a quick `!!` to cast as a boolean. We also don't actually need the `source: "app"` here because the store function adds it automatically (I know this was pre-existing) - so this could just be:
```js
builderStore.actions.analyticsPing({
  embedded: !!$appStore.embedded
})
```